community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/staging/TicketedProcessing.java

```diff
@@ -153,15 +153,24 @@ public void submit( FROM job )
     {
         return future( () ->
         {
-            while ( input.hasNext() )
+            try
```

Comment:
This was an existing problem that was exposed by doing the other change, so had to be fixed in order for this PR to go green